apps/passport-server/src/services/generic-issuance/pipelines/CSVPipeline/makeTicketPCD.ts

```diff
@@ -0,0 +1,49 @@
+import { EdDSATicketPCDPackage, TicketCategory } from "@pcd/eddsa-ticket-pcd";
+import { ArgumentTypeName, SerializedPCD } from "@pcd/pcd-types";
+import { randomUUID } from "crypto";
+import { v4 as uuid } from "uuid";
+import { traced } from "../../../telemetryService";
+
+export async function makeTicketPCD(
+  inputRow: string[],
+  eddsaPrivateKey: string
+): Promise<SerializedPCD> {
+  return traced("", "makeEdDSAMessageCSVPCD", async () => {
+    const pcd = await EdDSATicketPCDPackage.prove({
+      id: {
+        argumentType: ArgumentTypeName.String,
+        value: uuid()
+      },
+      privateKey: {
+        argumentType: ArgumentTypeName.String,
+        value: eddsaPrivateKey
+      },
+      ticket: {
+        argumentType: ArgumentTypeName.Object,
+        value: {
+          // The fields below are not signed and are used for display purposes.
+          eventName: inputRow[0],
+          ticketName: inputRow[1],
+          checkerEmail: undefined, // change if checkin feature enabled for csv pipelines
+          imageUrl: undefined,
```

Comment:
reminder to make this configurable, as probably all non-scannable badges, e.g. day passes, will have an image URL